docs/_documentations/troubleshooting.md

```diff
@@ -773,3 +774,31 @@ Info added in 0.10.0.
 To assist with problem determination, raise the default Codewind server log level to **Debug** or **Trace**. Use the `cwctl loglevels` command or follow the instructions for an IDE:
 - In Eclipse, enable support features in the Codewind preferences, then right-click the connection in the Codewind Explorer view and click **Codewind server log level**.
 - In VS Code, use the **Codewind: Set Codewind Server Logging Level** command in the Command Palette.
+
+# Executable file not found on PATH in VS Code
+<!--
+Action/Topic: Executable file not found on PATH in VS Code
+Issue type: info
+Issue link: https://github.com/eclipse/codewind/issues/1682
+Solution: https://github.com/eclipse/codewind/issues/1682#issuecomment-572649328
+Info added in 0.11.0.
+-->
+Codewind will not work if it cannot find the `docker` executable in the [`PATH`](https://en.wikipedia.org/wiki/PATH_(variable)). You will see an error message like the following:
```

Comment:
- Please keep in the present tense: "Codewind does not work if it..." and "You see an error message..."
- I'd prefer to link to a more "official" source than Wikipedia. Could we just say "the `PATH` variable" and remove the link? I tried an internet search for "variable path," and the Wikipedia entry came up as the second result, so I think folks should be OK.